setup.py

```diff
@@ -42,5 +42,5 @@
         "SQLAlchemy",
         "singledispatch>=3.4.0.3",
     ],
-    tests_require=["pytest>=2.7.2", "mock", "sqlalchemy_utils"],
+    tests_require=["pytest>=2.7.2", "mock", "sqlalchemy_utils", "enum34"],
```

Comment:
FYI this is needed to run test with python 2.7. I was not able to run the test with python 3 with and without this.

Reply:
I ended up removing it because of https://github.com/graphql-python/graphene-sqlalchemy/pull/177 already fixes it.